datahub-graphql-api/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -0,0 +1,87 @@
+package com.linkedin.datahub.graphql;
+
+import com.google.common.collect.ImmutableMap;
+import com.linkedin.datahub.graphql.loaders.CorpUserLoader;
+import com.linkedin.datahub.graphql.loaders.DatasetLoader;
+import com.linkedin.datahub.graphql.loaders.GmsClientFactory;
+import com.linkedin.datahub.graphql.resolvers.corpuser.ManagerResolver;
+import com.linkedin.datahub.graphql.resolvers.ownership.OwnerResolver;
+import com.linkedin.datahub.graphql.resolvers.query.DatasetResolver;
+import graphql.schema.idl.RuntimeWiring;
+import org.apache.commons.io.IOUtils;
+import org.dataloader.DataLoader;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import static com.linkedin.datahub.graphql.Constants.*;
+
+/**
+ * A {@link GraphQLEngine} configured to provide access to the
+ * entities and aspects on the the GMS graph.
+ */
+public class GmsGraphQLEngine {
+
+    public static GraphQLEngine _engine;
```

Comment:
nit: replace with static method to get back singleton?

Reply:
Should not be